web/themes/custom/cloudy/src/components/_header.scss

```diff
@@ -227,101 +228,131 @@
+      background-color: $cloudy-color-neutral-0 !important;
+    }
   }
 
-  // Header region - /node - !/front, !/search, !/search/*, !/404
+  // Header region - /node - !/front, !/search, !/search/*
   &--small {
     @include media-breakpoint-up(md) {
       margin: 0 $cloudy-space-4;
-    }
-    form {
-      flex-direction: row-reverse;
-    }
-    .cloudy-search-form__input {
-      border-radius: $border-radius-sm;
-    }
-    .cloudy-btn--icon[type="submit"] {
-      padding: $cloudy-space-0 $cloudy-space-5;
-      &:hover {
-        background-color: transparent !important;
+      form {
+        .form-actions {
+          background-color: $cloudy-color-neutral-200;
+          &:hover {
+            background-color: $cloudy-color-neutral-100;
+          }
+        }
+      }
+      .cloudy-btn--icon[type="submit"] {
+        background-color: $cloudy-color-neutral-900;
+        &:hover {
+          background-color: $cloudy-color-neutral-900 !important;
+        }
       }
     }
   }
-  // Content region - /front, /search, /search/*, /404
+  // Content region - /front, /search, /search/*
   &--large {
-    border-bottom: $search-large-border-style-underline;
-    border-radius: $cloudy-space-0 $cloudy-space-0 $border-radius-sm $border-radius-sm;
     form {
       height: 67px;
-      border-radius: $border-radius-sm $border-radius-sm $cloudy-space-0 $cloudy-space-0;
       margin: $cloudy-space-0;
-      border-top: $search-large-border-style;
-      border-right: $search-large-border-style;
-      border-left: $search-large-border-style;
+      border-radius: $border-radius-xl;
       @include media-breakpoint-up(md) {
         height: 69px;
       }
+      .form-actions {
+        background-color: $cloudy-color-blue-600;
+        border-radius: 0 $border-radius-md $border-radius-md 0;
+        &:hover {
+          background-color: $cloudy-color-blue-500;
+        }
+      }
     }
     .cloudy-search-form__input {
+      height: 100%;
+      width: 100%;
       padding: $cloudy-space-0 $cloudy-space-4;
       font-size: ($cloudy-lead-md-font-size / $cloudy-type-base-font-size) + rem;
-      border-radius: $border-radius-sm $cloudy-space-0 $cloudy-space-0;
-      height: 100%;
+      background-color: $cloudy-color-neutral-200;
     }
     .cloudy-btn--icon[type="submit"] {
-      border-radius: $cloudy-space-0 $border-radius-sm $cloudy-space-0 $cloudy-space-0;
-      background-size: 18px;
-      padding: $cloudy-space-0 $cloudy-space-6;
-      @include media-breakpoint-up(md) {
-        background-size: 20px;
+      padding: $cloudy-space-0 2.25rem;
+      -webkit-mask-size: 20px;
+      mask-size: 20px;
+      background-color: $cloudy-color-neutral-0;
+      &:hover {
+        background-color: $cloudy-color-neutral-0 !important;
       }
     }
   }
 }
 
+
 // TODO: Override module library with libraries-override
 // Search API block autocomplete
-.ui-autocomplete {
+.ui-autocomplete.search-api-autocomplete-search {
```

Comment:
.ui-autocomplete is also used by the entity reference field autocomplete. This change breaks those fields.